docs/musig2.md

```diff
@@ -42,7 +42,7 @@ to test the RPCs and to showcase the different use cases.
 ### 3-of-3 Taproot key spend path (BIP-0086)
 
 See `testTaprootMuSig2KeySpendBip86` in
-[`lntest/itest/lnd_taproot_test.go`](../lntest/itest/lnd_taproot_test.go) to see
+[`itest/lnd_taproot_test.go`](../itest/lnd_taproot_test.go) to see
```

Comment:
There's one more occurrence in `docs/code_contribution_guidelines.md` on line 158.

Reply:
fixed!